operations/routingrules.md

```diff
@@ -0,0 +1,380 @@
+| `ServiceId` | [String update value](#string-update-value) | required | Unique identifier of the [Service](services.md#service) routing rule is assigned to \(or `null` if should not be updated\). |
+| `RouteType` | [String update value](#string-update-value) | required | What should be routed [Route type](routingrules.md#route-type) \(or `null` if should not be updated\). |
+| `SelectedStayItems` | [Selected stay items update parameters](routingrules.md#selected-stay-items-update-parameters) | optional | To which stay items routing rule applies to. Returns only if RouteType value is SelectedStayItems. |
+
+#### String update value
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Value` | string | optional | Value which is to be updated. |
+
+#### String array update value
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Value` | array of string | optional | Value which is to be updated. |
+
+#### Bool update value
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Value` | bool | optional | Value which is to be updated. |
+
+#### Selected stay items update parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Nights` | [Bool update value](#bool-update-value) | optional | Whether routing rule applies to nights \(or `null` if should not be updated\). |
+| `CityTax` | [Bool update value](#bool-update-value) | optional | Whether routing rule applies to city tax \(or `null` if should not be updated\). |
+| `ProductCategoryIds` | [String array update value](#string-array-update-value) | optional | Product categories to which routing rule applies to \(or `null` if should not be updated\). |
+
+### Response
+
+```javascript
+{
+    "RoutingRules": [
+        {
+            "Id": "ff785b22-5422-4d1d-87f4-af2e00b3dfda",
+            "CompanyId": "35f493d1-6cf7-4132-982a-a5540549c322",
+            "CompanyRelation": "PartnerCompany",
+            "ServiceId": "708b3509-69ad-4a92-841b-d81f103edcee",
+            "Applicability": "Always",
+            "RouteType": "AllStayItems"
+        },
+        {
+            "Id": "d98c9611-0006-4691-a835-af2e00b170c4",
+            "CompanyId": "73a1fc8e-11c5-4019-b296-9234b651fc68",
+            "CompanyRelation": "TravelAgency",
+            "ServiceId": "708b3509-69ad-4a92-841b-d81f103edcee",
+            "Applicability": "Always",
+            "RouteType": "SelectedStayItems",
+            "SelectedStayItems":
+            {
+                "Nights" : false,
+                "CityTax" : true,
+                "ProductCategoryIds":[
+                    "004fa262-7b08-4853-b544-af2a00d5cf1f"
+                ]
+            }
+        }
+    ],
+    "Cursor": null
```

Comment:
Maybe we should delete this cursor. Even though the request shows it, it is not relevant in a context of update.